<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLogin</class>
 <widget class="QDialog" name="DialogLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QPushButton" name="b_Login">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Email">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Passw">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
